docs/troubleshoot/installation_overview.md

```diff
@@ -0,0 +1,117 @@
+# Percona Everest installation and workflow
+
+This section provides an overview of how Percona Everest is installed, the components involved, and the workflow, from operator installation to database provisioning, backups, and restores.
+
+
+## Installation workflow
+
+Starting with Percona Everest v1.4.0, the [CLI installation](../install/installEverest.md) is a wrapper around two helm charts:
+
+- [everest-core](https://github.com/percona/percona-helm-charts/tree/main/charts/everest){:target="_blank"} 
+
+- [everest-db-namespace](https://github.com/percona/percona-helm-charts/tree/main/charts/everest/charts/everest-db-namespace){:target="_blank"}. 
+
+The installation flow is as follows:
+{.power-number}
+
+1. The `everest-core` helm chart is installed in the `everest-system` namespace:
+
+    This step deploys all the core components required for Percona Everest to function.
+
+    | Component category   | Namespace            | Components                                                  |
+    |----------------------|----------------------|-------------------------------------------------------------|
+    | OLM components       | `everest-olm`        | `olm-operator`, `catalog-operator`, `package-server`, `everest-catalog` |
+    | Monitoring stack     | `everest-monitoring` | `vm-operator`, `kube-state-metrics`                         |
+    | Everest components   | `everest-system`     | `everest-server`, `everest-operator`                        |
+
+
+2. The `everest-db-namespace` chart is installed in the `everest` namespace. This chart handles the deployment of database operators using the following workflow:
+
+    1. Creates OLM subscriptions for each of the selected database operators (by default, all supported operators are selected).
+    2. OLM reconciles the subscriptions and creates an install plan for each operator.
+    3. A Helm hook creates the `everest-operators-installer` job that waits for the OLM install plan to be created and approves it once created.
```

Comment:
What does "it" mean in this sentence? The job or the plan. Avoid using ambiguous words.

Reply:
Plan